crates/router/src/analytics.rs

```diff
@@ -2025,7 +2025,7 @@ pub mod routes {
             },
             &auth::JWTAuth {
                 permission: Permission::Analytics,
-                minimum_entity_level: EntityType::Merchant,
+                minimum_entity_level: EntityType::Profile,
```

Comment:
Are the `/search` APIs at Profile Level now? 
Show there be separate APIs for Org Level and Merchant Level?

Reply:
Search will be a single API, it behaves differently based on the AuthInfo. And AuthInfo constructed will be affected based on the role of the user.

Reply:
This is not the case in other APIs because a org level user can switch between profile, merchant and org level views. But the search doesn't need to be like that.